dev/io.openliberty.grpc.1.0.internal.common/original.bnd

```diff
@@ -1,14 +1,11 @@
 #*******************************************************************************
-# Copyright (c) 2022 IBM Corporation and others.
+# Copyright (c) 2022, 2023 IBM Corporation and others.
```

Comment:
No need to update the date range if all you're doing is cleaning up the contributors in the header